### PR TITLE
adds the vali claymore

### PR DESCRIPTION
**Pull request by 567Turtle**
## About The Pull Request

Adds the vali claymore, deals 90 damage and takes 20 ticks to swing. Has to be held with two hands for full effect.

image

## Why It's Good For The Game

Kuro asked me too, the sprites look cool and more choices are always nice.

## Changelog
:cl:
add: Harvester claymore, available at marine vendors
/:cl:

### CODE REVIEW

File: code/game/objects/items/storage/holsters.dm [hunk 1]
```diff
@@ -234,6 +234,24 @@
...
+		/obj/item/weapon/twohanded/glaive/harvester,
+	)
+	can_hold = list(
+		/obj/item/weapon/twohanded/glaive/harvester,
+	)
+
+/obj/item/storage/holster/blade/harvester_claymore/full/Initialize()
+	. = ..()
+	var/obj/item/new_item = new /obj/item/weapon/twohanded/glaive/harvester(src)
+	INVOKE_ASYNC(src, .proc/handle_item_insertion, new_item)
```
Comment (ivanmixo) [inline code/game/objects/items/storage/holsters.dm:253]:
```suggestion
	INVOKE_ASYNC(src, PROC_REF(handle_item_insertion), new_item)
```
515

**Comment by Oscari-pog** [comment]
I was fully expecting a vali claymore mine

**Comment by iforgotmeword** [comment]
Sadly I thought the same I was about to so happy about bootleg OT 

(Still really mad they removed bootleg OT

**Comment by 567Turtle** [comment]
> This should use the other sprites that cs - m made, also image uhhhhhhhhhhhhhhhhhhhhhhhh

I have no clue how or why the other sprites there got deleted I'll fix it later and add the new spdites

**Comment by zzurge** [comment]
One step closer to crescent rose...

**Review by Kuro020 — APPROVED** [review]

**Comment by Jalopkomit18** [comment]
Everyday we grow closer to monster hunter. Add hunting horn next

File: code/game/objects/items/weapons/twohanded.dm [hunk 2]
```diff
@@ -345,6 +345,47 @@
...
+	force_wielded = 90
+	throwforce = 65
+	throw_speed = 3
+	edge = 1
+	attack_speed = 20
+	sharp = IS_SHARP_ITEM_BIG
+	w_class = WEIGHT_CLASS_BULKY
+	flags_item = DRAINS_XENO | TWOHANDED
+
+	var/codex_info = {"<b>Reagent info:</b><BR>
```
Comment (TiviPlus) [inline code/game/objects/items/weapons/twohanded.dm:363]: Autodoc

**Review by TiviPlus — CHANGES_REQUESTED** [review]
rest lgtm

**Comment by Al-1ce** [comment]
Sprites are a bit weird. The wire/connecting thing really screws up the perspective and makes it look less like a sword and more like a green 2x4

**Comment by chizzy376** [comment]
The green handheld plank

**Comment by 567Turtle** [comment]
> Sprites are a bit weird. The wire/connecting thing really screws up the perspective and makes it look less like a sword and more like a green 2x4

Send any sprite issues you have to Lactose as I can't sprite for shit and they're the ones who made it thank you

**Comment by docRoboRobert** [comment]
Totally not made for vergili.

**Comment by Lumipharon** [comment]
autodoc is 3 forward slashes

**Comment by 567Turtle** [comment]
autodoc is 3 slashes meh meh meh

File: code/game/objects/items/weapons/twohanded.dm [hunk 3]
```diff
@@ -345,6 +345,48 @@
...
+	throwforce = 65
+	throw_speed = 3
+	edge = 1
+	attack_speed = 20
+	sharp = IS_SHARP_ITEM_BIG
+	w_class = WEIGHT_CLASS_BULKY
+	flags_item = DRAINS_XENO | TWOHANDED
+
+/// Lists the information in the codex
+	var/codex_info = {"<b>Reagent info:</b><BR>
```
Comment (TiviPlus) [inline code/game/objects/items/weapons/twohanded.dm:364]:
```suggestion
	/// Lists the information in the codex
	var/codex_info = {"<b>Reagent info:</b><BR>
```

**Comment by Al-1ce** [comment]
In after sight this is pretty unbalanced.

**Pull request merged**